src/main/resources/templates/Advisor/insightsEmailBody.html

```diff
@@ -0,0 +1,606 @@
+      background-color: #c9190b;
+      background: #c9190b !important;
+      color: #ffffff;
+    }
+
+    .rh-card.rh-m-danger .rh-card__header h2 {
+      color: #FFFFFF;
+      color: #FFFFFF !important;
+    }
+
+    .rh-card.rh-m-no-danger {
+      background-color: #b8bbbe;
+      background: #b8bbbe !important;
+      color: #151515;
+    }
+
+    .rh-card.rh-m-no-danger svg,
+    .rh-card.rh-m-no-danger use {
+      fill: #151515;
+    }
+
+    .rh-card.rh-m-set-height {
+      height: 160px;
+    }
+
+    .rh-card-metrics .rh-card__header {
+      padding-bottom: 4px;
+    }
+
+    .rh-card__header h2 {
+      font-size: 14px;
+      font-weight: 500;
+    }
+
+    .rh-50-block {
+      width: 100%;
+      max-width: 281px;
+      display: inline-block;
+      vertical-align: top;
+      direction: ltr;
+    }
+
+    @media only screen and (max-width: 600px) {
+      body{
+        width: 100% !important;
+        min-width: 100% !important;
+      }
+
+      .rh-template-container {
+        max-width: 600px !important;
+        /*@editable*/ width: 100% !important;
+      }
+
+      .rh-body.rh-multi-column {
+        padding-bottom: 0;
+      }
+
+      .rh-50-block {
+        width: 100%;
+        max-width: 100% !important;
+        padding-bottom: 24px;
+      }
+
+      .rh-icon-wrapper {
+        width: 30px;
+        height: 48px;
+      }
+
+      .rh-card.rh-m-set-height {
+        height: auto;
+      }
+    }
+
+    /* Dark Mode */
+		/* @media (prefers-color-scheme: dark) {
+		      .rh-content .rh-content__block,
+		      .rh-card {
+		        background-color: #212427;
+		      }
+
+		      .rh-template-container,
+		      .rh-template-container > tbody {
+		        background-color: #3c3f42;
+		        color: #ffffff;
+		      }
+
+		      .rh-content a,
+		      .rh-footer__text a {
+		        color: #73bcf7;
+		      }
+
+		      .rh-cta-link a {
+		        color: #ffffff;
+		      }
+
+		      .rh-card.rh-m-danger {
+		        background-color: #c9190b;
+		        background: #c9190b;
+		        color: #ffffff;
+		      }
+
+		      .rh-card.rh-m-no-danger {
+		        background-color: #b8bbbe;
+		        background: #b8bbbe;
+		        color: #151515;
+		      }
+		    }
+
+		    [data‐ogsc] {
+		      .rh-content .rh-content__block,
+		      .rh-card {
+		        background-color: #212427;
+		      }
+
+		      .rh-template-container,
+		      .rh-template-container > tbody {
+		        background-color: #3c3f42;
+		        color: #ffffff;
+		      }
+
+		      .rh-content a,
+		      .rh-footer__text a {
+		        color: #73bcf7;
+		      }
+
+		      .rh-cta-link a {
+		        color: #ffffff;
+		      }
+
+		      .rh-card.rh-m-danger {
+		        background-color: #c9190b;
+		        background: #c9190b;
+		        color: #ffffff;
+		      }
+
+		      .rh-card.rh-m-no-danger {
+		        background-color: #b8bbbe;
+		        background: #b8bbbe;
+		        color: #151515;
+		      }
+		    }  */
```

Comment:
There are a few commented CSS rules like this one in the file. Is that OK to commit them?

Reply:
Ah... It doesn't look like they would affect the output in any way... I'll ask if these serve any purpose.

Reply:
That's harmless indeed.

Reply:
I'm going to leave this for now, but I asked if this is going to be enabled or if we can remove it